lib/reporters/mini.js

```diff
@@ -233,7 +233,8 @@ class MiniReporter {
 		}
 
 		if (runStatus.failFastEnabled === true && runStatus.remainingCount > 0 && runStatus.failCount > 0) {
-			status += '\n\n  ' + colors.information('`--fail-fast` is on. Any number of tests may have been skipped');
+			const remaining = runStatus.remainingCount + ' skipped ' + plur('test', runStatus.remainingCount) + '.';
```

Comment:
We don't necessarily know how many tests would have executed when running in `--concurrency` mode. So to be accurate this should be `At least 1 test was skipped.`.